Library/ViewModels/SettingsPrivacyViewModel.swift

```diff
@@ -72,10 +73,9 @@ SettingsPrivacyViewModelInputs, SettingsPrivacyViewModelOutputs {
       .takeWhen(self.unableToSaveError)
       .map { previous, _ in previous }
 
-   self.updateCurrentUser = Signal.merge(initialUser, updatedUser, previousUserOnError)
+   self.updateCurrentUser = Signal.merge(updatedFetchedUser, previousUserOnError)
```

Comment:
I removed the `initialUser` signal here to reduce the number of notification calls to `.ksr_userUpdated` which actually has a lot of listeners.